law.cfg

```diff
@@ -47,11 +47,13 @@ chunked_io_pool_size: 2
 chunked_io_debug: False
 
 # csv list of task families that inherit from ChunkedReaderMixin and whose output arrays should be
-# checked for non-finite values before saving them to disk (right now, supported tasks are
-# cf.CalibrateEvents, cf.SelectEvents, cf.ProduceColumns, cf.PrepareMLEvents, cf.MLEvaluation,
-# cf.UniteColumns)
+# checked (raising an exception) for non-finite values before saving them to disk
```

Comment:
Similar comment as in `cms_minimal` law config

Reply:
Here we should indeed keep list of tasks. Added back :+1: